routemaster/client/connection.rb

```diff
@@ -43,7 +43,7 @@ def subscribe(options)
           end
 
           unless response.success?
-            raise 'subscribe rejected'
+            raise "subscribe rejected (#{response.status})"
```

Comment:
maybe we can do a proper exception class here, so it's tracked away from your garden variety `RuntimeError`'s in New Relic?